modules/core/04-channel/keeper/keeper_test.go

```diff
@@ -506,7 +506,7 @@ func (suite *KeeperTestSuite) TestParams() {
 	}{
 		{"success: set default params", types.DefaultParams(), true},
 		{"success: zero timeout height", types.NewParams(types.NewTimeout(clienttypes.ZeroHeight(), 10000)), true},
-		{"success: zero timestamp timestamp", types.NewParams(types.NewTimeout(types.DefaultTimeout.Height, 0)), true},
+		{"fail: zero timeout timestamp", types.NewParams(types.NewTimeout(types.DefaultTimeout.Height, 0)), false},
```

Comment:
`types.DefaultTimeout.Height` is zero, so should be changed to a non-zero height